docs/modules/ROOT/pages/how-tos/storage/change-luks-key.adoc

```diff
@@ -0,0 +1,62 @@
+= Change LUKS Key on an Encrypted Rook Managed Ceph OSD
+
+This page describes how to change the LUKS encryption key of an OSD backing storage when using Rook to manage the Ceph cluster.
+
+== Information Gathering
+
```

Comment:
To make the how-to a bit easier to follow, we probably want to provide environment variables for the Ceph cluster namespace and the OSD pod name.

```suggestion

. Export namespace of Ceph cluster, we'll use this in later steps
+
----
export CEPHCLUSTER_NAMESPACE=rook-ceph
export OSDID=X # select which OSD you want to operate on
export OSD_POD=$(kubectl -n "${CEPHCLUSTER_NAMESPACE}" get pods -l ceph-osd-id="${OSDID}" -o jsonpath='{.items[0].metadata.name}')
----
```